.github/workflows/good-first-issue.yml

```
@@ -26,7 +26,7 @@ jobs:
             the sidebar for linked pull requests (PRs). If there are none, feel
             free to start working on it. If there is an open PR, please
             collaborate on the work by reviewing it rather than
-            duplicating it in a competeing PR.
+            duplicating it in a competing PR.
 
 
             If something is confusing or unclear, please reach out on any of our
```

While at it, we can simplify here:

```suggestion
            If something is unclear, please reach out on any of our
```

I see confusing as a sub-category of unclear. Unclear should be sufficient.